### PR TITLE
Fix all the mypy issues

### PR DESCRIPTION
**Pull request by dragon-dxw**
Ensure that we are typechecking things.

### CODE REVIEW

**Comment by dragon-dxw** [comment]
TODO: fix `judgements` folder and all references thereof

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -64,6 +64,9 @@ jobs:
       - name: Generate coverage XML
         run: docker-compose run django coverage xml
 
+      - name: Run mypy
+        run: docker-compose run django mypy ds_judgements_public_ui judgments
```
Comment (dragon-dxw) [inline .github/workflows/ci.yml:68]: we need to fix this `judgements` :)

File: fabfile.py [hunk 2]
```diff
@@ -122,6 +122,7 @@ def test(c):
             "django",
             "mypy",
             "ds_judgements_public_ui",
+            "judgments",
```
Comment (dragon-dxw) [inline fabfile.py:125]: check the other directory too when running `fab test`

File: judgments/views/detail.py [hunk 3]
```diff
@@ -113,15 +110,17 @@ def detail_xml(_request, judgment_uri):
...
+        headers={"Accept-Encoding": None},  # type: ignore
     )
     content_length = response.headers.get("Content-Length", None)
     if response.status_code >= 400:
         return ""
     if content_length:
         filesize = filesizeformat(int(content_length))
         return f" ({filesize})"
-    logging.warn(f"Unable to determine PDF size for {judgment_uri}")
+    logging.warning(f"Unable to determine PDF size for {judgment_uri}")
```
Comment (dragon-dxw) [inline judgments/views/detail.py:123]: "warn" is deprecated, according to error messages during pytest testing

**Review by jacksonj04 — APPROVED** [review]
Happy that although this doesn't make things perfect it makes them better (and helps prevent us being silly in future, which is more important).

**Pull request merged**